code/src/components/climbing-area/climbing-area.component.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+
+import AreaOption from "../area-option/area-option.component";
+import {
+  ButtonContainer,
+  ButtonText,
+  ButtonsContainer,
+  Label
+} from "../../assets/styles/shared";
+
+const areas = ["Indoor", "Outdoor"];
+
+const ClimbingArea = () => {
+  const [area, setArea] = useState(localStorage.getItem("area") || "");
+  const onNextHandler = (e) => {
+    e.preventDefault();
+    localStorage.setItem("area", area);
+  };
+
+  return (
+    <>
+      <Label>What type of area for climbing do you prefer?</Label>
+      {areas.map((climbingArea) => {
+        return (
+          <AreaOption
+            area={climbingArea}
+            setArea={setArea}
+            key={climbingArea} />
+        );
+      })}
```

Comment:
Nice! I really like this functions! both here and live in the site. Well done 🙋‍♀️